test/explorer/polars_backend/decimal_unstable_test.exs

```diff
@@ -0,0 +1,65 @@
+defmodule Explorer.PolarsBackend.DecimalUnstableTest do
+  # This module tracks some oddities we've found with decimal dtypes. The Polars
+  # docs warn that decimals are unstable:
+  #
+  #   > This functionality is considered unstable. It is a work-in-progress
+  #   > feature and may not always work as expected. It may be changed at any
+  #   > point without it being considered a breaking change.
+  #
+  # https://docs.pola.rs/api/python/stable/reference/api/polars.datatypes.Decimal.html
+  #
+  # If the tests in the module start breaking, it probably means Polars has
+  # changed its decimal implementation. Be prepared to change the tests: they
+  # function as canaries rather than imposing expected behavior.
+
+  use ExUnit.Case, async: true
+
+  alias Explorer.PolarsBackend
+
+  require Explorer.DataFrame, as: DF
+
+  setup do
+    %{df: DF.new(a: [Decimal.new("1.0"), Decimal.new("2.0")])}
+  end
+
+  test "mean returns decimal instead of float", %{df: df} do
+    assert_raise(
+      RuntimeError,
+      """
+      DataFrame mismatch.
+
+      expected:
+
+          names: ["a"]
+          dtypes: %{"a" => {:f, 64}}
+
+      got:
+
+          names: ["a"]
+          dtypes: %{"a" => {:decimal, 38, 1}}
+      """,
+      fn -> DF.summarise(df, a: mean(a)) end
+    )
+  end
```

Comment:
This is actually a result of our expectations:

https://github.com/elixir-explorer/explorer/blob/020fb23e6db145a640b3f1fcb1fe95f404afb8ee/lib/explorer/backend/lazy_series.ex#L772-L783

I'm not entirely sure it's correct.

Reply:
@josevalim Sorry forgot about this. Are you saying we should remove this test and force our decimal operations to return float instead?

Reply:
I think we should return whatever Polars return for now and we will fix it naturally as Polars fixes it.

Reply:
Ok, then the test can stay. When Polars fixes their end, we will learn about it because the test will break. Then we can convert it to a test of what the behavior should actually be.